quickstart/docker/image/run-router.sh

```diff
@@ -12,6 +12,7 @@ if [[ "${ZITI_EDGE_ROUTER_PORT-}" == "" ]]; then export ZITI_EDGE_ROUTER_PORT="3
 if [[ "${ZITI_EDGE_ROUTER_HOSTNAME}" == "" ]]; then export ZITI_EDGE_ROUTER_HOSTNAME="${ZITI_EDGE_ROUTER_RAWNAME}${ZITI_DOMAIN_SUFFIX}"; fi
 if [[ "${ZITI_EDGE_ROUTER_ROLES}" == "" ]]; then export ZITI_EDGE_ROUTER_ROLES="${ZITI_EDGE_ROUTER_RAWNAME}"; fi
 
+generateEnvFile
```

Comment:
not sure about this. i think we need to be careful here - this caused a race condition for the docker-compose environment. i had removed this for that reason.

Reply:
Yeah, I recall this being removed I thought it was a race condition between the controller and router causing the conflict so I had added the depends_on. I added this back because there was an env var not getting set in the controller startup that was needed. I don't recall off hand what that was but I'll take a look today and see if I can work around it and remove this again.